src/App.css

```diff
@@ -1,38 +1,133 @@
+/* CSS */
+
 .App {
-  text-align: center;
+  height: 600px;
+  overflow: scroll;
```

Comment:
you probably don't need this - the default will already be scrollable, and this will make a scrollbar always show up, which is a little ugly.

```suggestion
```